src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -88,6 +92,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
     }
     public dispose() {
         this.disposables.forEach((item) => item.dispose());
+        this.debugReadyInterpreters.clear();
```

Comment:
You should only need to worry about disposables here. This is just a Map, it's going away in any case.